Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Pages/FunctionPickerComponent.razor

```diff
@@ -15,7 +15,10 @@
         </div>
     </div>
     <div class="col-sm form-group">
-        <label class="col-xs-3 control-label" for="functions-picker">Function</label>
+        <div class="col-xs-5">
+            <label class="control-label" for="functions-picker">Function</label>
+            <button class="float-right btn btn-primary btn-sm" id="functions-picker-sort-order" @onclick="SortFunctionsClicked" >Sort Functions</button>
```

Comment:
Should we just always sort the functions? I'm not sure if there is a lot of value for having the option of seeing them list in the order defined in the template. This way you are going to have to click the sort button every time you start the the tool which will probably get old fast.